apps/client/src/features/editors/finder/useFinder.tsx

```diff
@@ -29,6 +29,12 @@ export default function useFinder() {
   const [results, setResults] = useState<FilterableEntry[]>([]);
   const [error, setError] = useState<MaybeString>(null);
 
+  /** clear results when source data changes */
+  useEffect(() => {
+    setResults([]);
```

Comment:
this clears out the result if you are searching while an update arrives
could we maybe hold the search query in state also and then rerun the search if the data changes

Reply:
good catch